### PR TITLE
Dep convergence

### PR DESCRIPTION
**Pull request by piotr921**
Check pom.xml.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,6 +8,43 @@
     <artifactId>Battleship</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <description>Java implementation of Battleship game (client-server architecture with simple GUI in JavaFX).
+        See game rules: https://en.wikipedia.org/wiki/Battleship_(game)
+    </description>
+    <developers>
+        <developer>
+            <name>Michał Partacz</name>
```
Comment (mprtcz) [inline pom.xml]: the MVP ❤️

File: pom.xml [hunk 2]
```diff
@@ -8,6 +8,43 @@
...
+                <role>Game Engineer</role>
+                <role>Architecture designer</role>
+                <role>Message bus driver</role>
+            </roles>
+        </developer>
+        <developer>
+            <name>Patrycja Zaremba</name>
+            <roles>
+                <role>Project Manager</role>
+                <role>Healthy food supplier</role>
```
Comment (mprtcz) [inline pom.xml]: 👍

File: pom.xml [hunk 3]
```diff
@@ -24,6 +61,7 @@
         <findbugs.version>3.0.4</findbugs.version>
         <checkstyle.version>2.17</checkstyle.version>
         <jacoco.version>0.7.9</jacoco.version>
+        <jxr.version>2.5</jxr.version>
```
Comment (mprtcz) [inline pom.xml]: 👍

File: pom.xml [hunk 4]
```diff
@@ -36,6 +74,11 @@
         <site.plugin.version>3.6</site.plugin.version>
         <surefire.plugin.version>2.20</surefire.plugin.version>
         <enforcer.plugin.version>1.4.1</enforcer.plugin.version>
+        <!--Sonar properties-->
+        <sonar.pluginClass>org.sonar.plugins.java.JavaPlugin</sonar.pluginClass>
+        <sonar.pluginName>Java</sonar.pluginName>
+        <!--<property name="sonar.java.libraries" location="${project.build.dir}/classes" />-->
```
Comment (mprtcz) [inline pom.xml]: Why the comments?

File: pom.xml [hunk 5]
```diff
@@ -59,6 +102,33 @@
...
+                            <report>distribution-management</report>
+                            <report>issue-tracking</report>
+                            <report>license</report>
+                            <report>mailing-list</report>
+                            <report>modules</report>
+                            <report>plugin-management</report>
+                            <report>plugins</report>
+                            <report>project-team</report>
+                            <report>scm</report>
+                            <report>summary</report>
```
Comment (mprtcz) [inline pom.xml]: Verbose :suspect:

File: src/main/java/com/spanish_inquisition/battleship/server/BattleshipServer.java [hunk 6]
```diff
@@ -9,15 +9,14 @@
...
-
 public class BattleshipServer {
     private static final Integer PORT_NUMBER = 6666;
     static final int NUMBER_OF_PLAYERS = 2;
     static List<ClientConnectionHandler> clients;
 
     public static void main(String[] args) {
         initializeLogger();
-        connectWithPlayers(createServerSocket());
+        connectWithPlayers(createServerSocket(PORT_NUMBER));
```
Comment (mprtcz) [inline src/main/java/com/spanish_inquisition/battleship/server/BattleshipServer.java]: Ok a welcome fix

File: pom.xml [hunk 7]
```diff
@@ -8,6 +8,43 @@
...
+            </roles>
+        </developer>
+        <developer>
+            <name>Patrycja Zaremba</name>
+            <roles>
+                <role>Project Manager</role>
+                <role>Healthy food supplier</role>
+            </roles>
+        </developer>
+    </developers>
```
Comment (Dzionek95) [inline pom.xml]: great markup!!

File: pom.xml (hunk 4 above)
Comment (Dzionek95) [inline pom.xml]: Is this comment necessary?

File: pom.xml [hunk 8]
```diff
@@ -8,6 +8,43 @@
     <artifactId>Battleship</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <description>Java implementation of Battleship game (client-server architecture with simple GUI in JavaFX).
+        See game rules: https://en.wikipedia.org/wiki/Battleship_(game)
+    </description>
```
Comment (kukulastudent) [inline pom.xml]: :+1:

File: pom.xml (hunk 2 above)
Comment (kukulastudent) [inline pom.xml]: roles :+1: :1st_place_medal:

File: src/main/java/com/spanish_inquisition/battleship/server/BattleshipServer.java [hunk 9]
```diff
@@ -9,15 +9,14 @@
...
     }
 
     static ServerSocket createServerSocket(int portNumber) {
```
Comment (kukulastudent) [inline src/main/java/com/spanish_inquisition/battleship/server/BattleshipServer.java]: Important: change to non static !

**Pull request merged**